docs/work/content/04_content/voraussetzungen.tex

```diff
@@ -1,15 +1,106 @@
 \chapter{Voraussetzungen}
 
-	\section{Machine Learning}
+%	\section{Machine Learning}
 	
-	\section{}
+	\section{RESTful APIs} % Raphael
+	
+		Representational State Transfer beschreibt ein architektonisches Modell, um Web Services zu erstellen. Sogenannte \ac{REST} Web Services bieten Interoperabilität zwischen Computersystemen im Internet -- geeignet für die Kommunikation von Maschine zu Maschine. So lässt sich REST auch als eine Abstraktion der Struktur und des Verhaltens des World Wide Webs beschreiben. 
+		
+		Neben REST gibt es weitere Alternativen, wie \ac{SOAP} oder \ac{RPC}; der Vorteil von REST besteht jedoch darin, dass durch das \ac{WWW} ein Großteil der Infrastruktur für die Kommunikation bereits vorhanden und implementiert ist. Während bei \\ac{RPC} in der \ac{URI} Methodeninformationen enthalten sind, gibt eine \ac{URI} in der \ac{REST} Architektur ausschließlich Ressourcen an und kodiert die Funktionalität mittels \ac{HTTP} Methoden. Dieser Ansatz entspricht dem Konzept einer \ac{URI}, da bei einer \ac{HTTP} Anfrage ebenso nur Ressourcen und keine Funktionalität gefragt ist.
+		
+		Eine REST API muss insgesamt sechs Eigenschaften besitzen:
+		
+		\begin{description}
+			\item[Client-Server Architektur] 
+				Bei \ac{REST} gilt im Allgemeinen, dass eine Client-Server Architektur herrschen soll: Die Client konsumiert die vom Server bereitgestellten Dienste. 
+			\item[Zustandslosigkeit] 
+				Eine RESTful \ac{API} soll keine Zustände haben, sondern so konzipiert werden, dass benötigten Informationen in einer REST-Nachricht vorhanden sind. Dies begünstigt außerdem die Skalierbarkeit eines solchen Dienstes, da auf diese Weise einfach alle Anfragen auf mehrere Instanzen verteilt werden können.
```

Comment:
[...] in einer REST-Nachicht enthalten sind [...]
[...] da es auf diese Weise sehr einfach ist, alle Anfragen.... zu verteilen[...]